lib/client/client.go

```diff
@@ -95,6 +97,13 @@ type NodeClient struct {
 	// addr set in TC.WebProxyAddr. This is needed to report the correct address
 	// to SSH_TELEPORT_WEBPROXY_ADDR used by some features like "teleport status".
 	ProxyPublicAddr string
+
+	// hostname is the node's hostname, for more user-friendly logging.
+	hostname string
+
+	// sshLogDir is the directory to log the output of multiple SSH commands to.
+	// If not set, no logs will be created.
+	sshLogDir string
```

Comment:
Is this only used when multiple nodes match the expression? If I run a command on a label that happens to only match a single node, do I get log files or no?

Reply:
As it is now, if only one node is matched the summary log files won't be written and the lines won't be labeled. The stdout and stderr files will be written for any number of nodes as long as `--log-dir` is present.